wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectService.java

```diff
@@ -351,8 +361,16 @@ public void importProject(@ApiParam(value = "Path in the project", required = tr
                                                                      ServerException,
                                                                      NotFoundException,
                                                                      BadRequestException {
-        projectManager.importProject(path, sourceStorage, force,
-                                     () -> new ProjectImportOutputWSLineConsumer(path, workspace, 300));
+
```

Comment:
I see that [issue](https://github.com/eclipse/che/issues/4591) description does not state this and that's a bit odd as that's the main idea here. We can create a separate issue for this if you think that this would require additional work or we can apply changed directly to this PR if that would take not much time, it is up to you.